### PR TITLE
Server migration

### PR DESCRIPTION
**Pull request by victor-paumier**

### CODE REVIEW

File: src/EventListener/LocaleListener.php [hunk 1]
```diff
@@ -81,11 +81,12 @@ public function onKernelRequest(GetResponseEvent $event)
             $request->setLocale($locale);
         }
 
-        $user = $this->tokenStorage->getToken()->getUser();
-        if ($user instanceof User) {
-            $user->setLocaleLanguage($locale);
-            $this->om->persist($user);
-            $this->om->flush();
+        if ($token = $this->tokenStorage->getToken() instanceof TokenStorageInterface) {
```
Comment (JeremieSamson) [inline src/EventListener/LocaleListener.php]: why in two lines ?

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        - traefik.enable=false
+
+    php:
+        environment:
+          maildev_host: sf4_maildev
+        depends_on:
+        - maildev
+        - mysql
+
+    maildev:
```
[the remark this answers is not in the document]
Comment (victor-paumier) [reply docker-compose.dev.yml]: docker-compose.dev.yml = docker-compose. override.yml

File: config/packages/twig.yaml [hunk 3]
```diff
@@ -5,5 +5,4 @@ twig:
     globals:
         fos_csrf_provider: "@security.csrf.token_manager"
         default_locale: '%locale%'
-        app_locales: '%app_locales%'
-        base_url: '%env(resolve:BASE_URL)%'
+        app_locales: '%app_locales%'
```
[the remark this answers is not in the document]
Comment (victor-paumier) [reply config/packages/twig.yaml]: done

File: src/DataFixtures/UserFixtures.php [hunk 4]
```diff
@@ -25,6 +25,7 @@ public function load(ObjectManager $manager)
 
         /** @var Language $lnEn */
         $lnEn = $this->getReference(LanguageFixtures::LANGUAGE_REFERENCE.'_en');
+        $locale = $lnEn->getCode();
```
[the remark this answers is not in the document]
Comment (victor-paumier) [reply src/DataFixtures/UserFixtures.php]: done

File: src/EventListener/LocaleListener.php (hunk 1 above)
Comment (victor-paumier) [reply src/EventListener/LocaleListener.php]: done

File: docker-compose.dev.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (JeremieSamson) [reply docker-compose.dev.yml]: better practice to have a docker-compose.override.yml.dist with everything commented then a non committed file (docker-compose.dev.yml or docker-compose. override.yml) on your specific env.

File: deploy.php [hunk 5]
```diff
@@ -26,53 +26,42 @@
...
-host('demo')
-    ->hostname('ftp.buddysystem.eu')
-    ->stage('demo')
+host('prod')
+    ->hostname('54.37.72.77')
+    ->stage('prod')
     ->set('symfony_env', 'prod')
     ->set('branch', 'master')
-    ->user('buddysysuj')
+    ->user('victor')
```
[the remark this answers is not in the document]
Comment (victor-paumier) [reply deploy.php]: done

File: deploy.php [hunk 6]
```diff
@@ -26,53 +26,42 @@
...
-    ->set('deploy_path', '/homez.717/buddysysuj/www/staging')
-    ->forwardAgent(true);
-
-host('prod')
-    ->hostname('ftp.buddysystem.eu')
-    ->stage('prod')
-    ->set('symfony_env', 'prod')
-    ->set('branch', 'master')
-    ->user('buddysysuj')
+    ->user('victor')
```
[the remark this answers is not in the document]
Comment (victor-paumier) [reply deploy.php]: done

**Pull request merged**